contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -379,7 +379,7 @@ CREATE OR REPLACE VIEW sys.sp_columns_100_view AS
      )
      , sys.translate_pg_type_to_tsql(a.atttypid) AS tsql_type_name
      , sys.spt_datatype_info_table AS t5
-  WHERE (t4."DATA_TYPE" = t5.TYPE_NAME COLLATE sys.database_default)
+  WHERE (CAST(t4."DATA_TYPE" AS varchar(20)) = t5.TYPE_NAME)
```

Comment:
This should be sys.varchar, right?

Reply:
Yes, should be sys.varchar

Reply:
`t5.TYPE_NAME` => `sys.spt_datatype_info_table.TYPE_NAME` is `pg_catalog.varchar` (charater varying)

Reply:
In that case, let's cast t5.TYPE_NAME to sys.nvarchar(128).

Reply:
Done